src/GridFieldOrderableRows.php

```diff
@@ -359,7 +360,15 @@ public function getManipulatedData(GridField $grid, SS_List $list)
                 // Fix bug in 3.1.3+ where ArrayList doesn't account for quotes
                 $sortterm .= $this->getSortTable($list).'.'.$this->getSortField();
             } else {
+                // If not an ArrayList it's definitely a DataList right?
```

Comment:
Nope. The param says `SS_List` (which is an interface) and there are more classes implementing it than just `ArrayList` and `DataList`. It's also extended to `Sortable` which may be implemented by yet another set of classes, so we can't assume the `->dataClass()` method is available without further checks.

Reply:
You're right! I missed the sub-interfaces of SS_List.

Happy to alter the PR. I'm thinking of just directly checking `method_exists($list, 'dataClass)` or do you think `$list instanceof DataList` would be cleaner?

Reply:
I would check for the `DataList` and only apply the default sort as the secondary ordering there

Reply:
Checking for a DataList now.